app/models/response.rb

```diff
@@ -51,4 +55,17 @@ def invitation
   def events
     Events::Event.where response_uuid: uuid
   end
+
+  def created_at
+    invitation_accepted.created_at
+  end
+
+  def invitation_accepted
+    Events::InvitationAccepted.find_by response_uuid: uuid
+  end
+
+  def self.find_by_show_secret(show_secret)
+    invitation_accepted = Events::InvitationAccepted.find_by_show_secret show_secret
+    find invitation_accepted.response_uuid
```

Comment:
Hier moet nog worden gekeken of er wel een ResponseFinished event bestaat met dezelfde response_uuid als het InvitationAccepted event. Anders kun je scores opvragen die niet door de invuller zijn bevestigd.